aspnetcore/performance/performance-best-practices/samples/3.0/Controllers/AsyncBadVoidController.cs

```diff
@@ -15,8 +15,6 @@ public async void Get()
         {
             await Task.Delay(1000);
 
-            // THIS will crash the process because it's writing after the response has 
```

Comment:
Why did you remove this comment?

Reply:
> Why did you remove this comment?


See my  notes below. We generally move all comments to the doc so they can be MT.

Reply:
Let me know which ones you want back and I can bend the rules and add a few.

Reply:
Edited and added back :)